packages/host/app/routes/card.gts

```diff
@@ -29,16 +32,29 @@ export default class RenderCard extends Route<Model | null> {
     operatorModeEnabled: {
       refreshModel: true,
     },
+    authMode: {
+      refreshModel: true,
+    },
+    sid: {
```

Comment:
Add comments what `sid` and `clientSecret` query params are for and where they come from